src/slider/examples.js

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+/* eslint-disable react/display-name*/
+
+import * as React from 'react';
+// Styled elements
+import {StatefulSlider as Slider} from './index';
+
+export const suite = 'Slider Test Suite';
+export const tests = {
```

Comment:
can you move this out to an `example-list.js` file, just like how the modal / chechbox has it?

Reply:
ok